models/stg_marketo__lead.sql

```diff
@@ -10,20 +10,21 @@ with leads as (
 
 ), merged_leads as (
 
-    select *
+    select 
+        cast(lead_id as {{ dbt_utils.type_int() }}) as lead_id,
+        -- in case there are multiple lead IDS merged at once, insert a space betweeen the comma and the following ID
+        replace( cast(merged_lead_id as {{ dbt_utils.type_string() }}), ',', ', ') as merged_lead_id
```

Comment:
stylistic choice 🤷‍♂️ i tend to prefer a space between commas, so this is to remain consistent with line 24